docs/tutorials/monero.md

```diff
@@ -0,0 +1,61 @@
+Monero
+==================
+
+## Connect to remote nodes.
+
+To connect Monero nodes, you have to configure the wallet software:
+
+In [Feather wallet](http://rwzulgcql2y3n6os2jhmhg6un2m33rylazfnzhf56likav47aylq.b32.i2p/): File -> Settings -> Network -> Proxy > i2p, with this option all network traffic will be routed over i2p, including websocket connection (if enabled).
+
+## Connection between other I2P nodes.
+
+1) Configure i2pd to create server type tunnel and put in your `./tunnels.conf` file:
+
+    [monero-node]
+    type = server
+    host = 127.0.0.1
+    port = 18080 # P2P port
```

Comment:
```suggestion
    port = 18085 # P2P port
```

Reply:
why different port needed?

Reply:
why different port used?

Reply:
18080 is the p2p port. Anonymous inbound uses custom/new ports